llvm/include/llvm/DebugInfo/DWARF/DWARFAcceleratorTable.h

```diff
@@ -793,6 +799,14 @@ class DWARFDebugNames : public DWARFAcceleratorTable {
   const NameIndex *getCUNameIndex(uint64_t CUOffset);
 };
 
+// Calculates the starting offsets for various sections within the
+// debug_names section. This has been pulled out of DWARFDebugNames, to
+// allow easier sharing of the code with other tools, such as LLD.
```

Comment:
///

Reply:
> This has been pulled out of DWARFDebugNames, to
 // allow easier sharing of the code with other tools, such as LLD.

This is more of a review comment than documentation, I suggest removing this

Reply:
Done.